### ISSUE TITLE
White screen when starting on windows 11 after update 1.7.3

### ISSUE THREAD

**Issue by Chupakabra0** [drakonkat/webtorrent-express-api#6]
**Describe the bug**
White screen when starting on windows 11

**Screenshots**

![img](https://user-images.githubusercontent.com/47476146/183080876-dc9d2cf7-8259-4a48-bc54-94c041b2de3f.png)

**Desktop:**
* OS: Windows 11
*  Version: 22H2

**Comment by Aidenww1** [comment]
same problem here

**Comment by drakonkat** [comment]
I noticed that the version 1.7.3 has a bug, i removed from the release list and added a version 1.7.4. 
Can you try to manually install it and see if the problem is solved?

[Download 1.7.4](https://github.com/drakonkat/Crawfish/releases/tag/v1.7.4)

Sorry for the inconvenience

**Comment by Chupakabra0** [comment]
Alas, I have this kind of bug even on 1.7.4 version. White screen, mouse plays loading animation but nothing happens even after 10 minutes of waiting.

**Comment by drakonkat** [comment]
Ok, i will try to reproduce the bug

**Comment by drakonkat** [comment]
If possible, it can be useful if you run the software from Powershell/CMD, it will print some log

To run it you need to go to the directory of the software (Usually "C:\Users\<<USERNAME>>\AppData\Local\Programs\CrawFish") and the run ".\Crawfish.exe" then paste here the logs

In any case I will try to add an automatic "log sender" if some error will happen

**Comment by Chupakabra0** [comment]
No problem. here's [logs](https://pastebin.com/Z21ZLZX4)
Interesting, some resource check has failed.

**Comment by drakonkat** [comment]
Ok, thanks a lot the problem is here:
```
*** uncaughtException: TypeError: Cannot read properties of undefined (reading 'download')
    at C:\Users\roota\AppData\Local\Programs\CrawFish\resources\app.asar\routes\classes\ConfigStorage.js:110:44
    at <anonymous>
    at process.processTicksAndRejections (node:internal/process/task_queues:96:5)
```
The resource failure is "ok", it's part of the starting process where i will find the best "searx" implementation which i need after to search torrent. It will fetch all the public searx server and save the best response on the client PC
    

I have introduced alternative speed limit, but it will make a wrong check i will fix it and try to release it today (It will happen only if before the versione 1.7.4 you never changed the upload/download limit i suppose and this is why i cannot replicate)

Thanks a lot for the help

**Comment by Chupakabra0** [comment]
Thank you for quick answer.

Also I left the application loading in the background for a while and the following appeared in the logs `[7260:0808/112540.202:ERROR:gpu_init.cc(481)] Passthrough is not supported, GL is disabled, ANGLE is`

Is it ok?

**Comment by drakonkat** [comment]
Hi, no but probably the first error cause rendering problem for electron i hope will be fixed with the first bug

**Comment by drakonkat** [comment]
Hi, it took some time, but now I have released the fix, with the version 1.7.5 I think you have no more problem (I think I will add some rollback options from update)

**Comment by Chupakabra0** [comment]
Works fine, thank you!
